tasks/feature-flag.yml

```
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2023 Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: 2023 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
+- name: "Check if a feature flag is already enabled"
```

I'd suggest to append the feature flag name to the task name to be able to identify which feature flag is handled:

```suggestion
- name: "Check if feature flag is already enabled for {{ gitlab_feature_flag.name }}"
```